console/server/src/main/java/quarks/console/server/ServerUtil.java

```diff
@@ -45,31 +53,35 @@ private String getPath() {
      * @return a File object
      */
     private File getTopDirFilePath() {
+        String topDirProp = System.getProperty("top.dir.file.path");
```

Comment:
~nit: can we use a property name that's scoped by "quarks"?  It seems this new property is required/used only to make things work in the build/test context.  If that's true then perhaps also have the name capture that: "quarks.test. ..."?